src/screens/auth/register/register.screen.tsx

```diff
@@ -11,38 +13,85 @@ import {
   TermsContainer,
   StyledCheckbox,
   StyledLink,
-} from './register.screen.styles';
+} from "./register.screen.styles";
+import { useServices } from "services";
+import { useStores } from "store";
+import { RoutePath } from "navigation/route-path";
+
+export const RegisterScreen = observer(() => {
+
+
+  const { accountService } = useServices();
+  const { accountStore } = useStores();
+  let history = useHistory();
+  const [registering, setRegistering] = useState(false);
+
+  const [fullName, setFullName] = useState("");
+  const [email, setEmail] = useState("");
+
+  const register = async () => {
+    try {
+      accountStore.setFetching(true)
+      const account = await accountService.register(fullName, email);
+      if (account.hasData()) {
+        history.push(RoutePath.home);
+      } else {
+        console.log(account.getErrorMessage());
+        history.push(RoutePath.register);
+      }
+      accountStore.setFetching(false);
+    } catch (e) {
+      console.log(e);
+    }
+  };
 
-export const RegisterScreen = () => {
   return (
     <LoginContainer>
       <Header />
 
       <LoginFormContainer>
         <FormContainer>
-          <LoginText variant='contentHeader' center tx='auth.createAccount' />
+          <Alert variant="error">
+            We couldn't find your account. Please register here.
+          </Alert>
+          <LoginText variant="contentHeader" center tx="auth.createAccount" />
 
-          <Box style={{ gap: '2.6rem' }}>
-            <Input type='text' label='Enter your Full Name' placeholder='John Doe' />
-            <Input type='text' label='Enter your Email' placeholder='johndoe@example.com' />
+          <Box style={{ gap: "2.6rem" }}>
+            <Input
+              type="text"
+              label="Enter your Full Name"
+              placeholder="John Doe"
+              onChange={(e: any) => {
+                setFullName(e.target.value);
+              }}
+            />
+            <Input
+              type="text"
+              label="Enter your Email"
+              placeholder="johndoe@example.com"
+              onChange={(e: any) => setEmail(e.target.value)}
+            />
           </Box>
 
           <TermsContainer>
-            <StyledCheckbox type='checkbox' />
-            <Text variant='content'>
-              By clicking, you agree to <StyledLink to={'safient.io'}>Safient's Terms </StyledLink>
-              of Use & <StyledLink to={'safient.io'}> Privacy Policy.</StyledLink>
+            <StyledCheckbox type="checkbox" />
+            <Text variant="content">
+              By clicking, you agree to{" "}
```

Comment:
If we are supporting localization, then the strings should be added to that file.